packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -335,6 +407,52 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
               />
             </section>
           )}
+
+          {config.showServiceQueueFields && (
+            <>
+              <section className={styles.section}>
+                <div className={styles.sectionTitle}>{t('service', 'Service')}</div>
```

Comment:
It'd be nice in the future to have an empty state here for situations where services aren't configured.